client/state/reader/streams/selectors/get-reader-stream-transformed-items.ts

```diff
@@ -17,12 +17,13 @@ import 'calypso/state/reader/init';
  * function( state, { streamKey: string, recsStreamKey: string }): Array
  */
 export const getTransformedStreamItems = treeSelect(
-	( state, { streamKey, recsStreamKey } ) => [
+	( state, { streamKey, recsStreamKey }: { streamKey: string; recsStreamKey: string } ) => [
```

Comment:
Necessary only to complete the type definition. Went the simple inline way, no need for abstractions IMHO.